### ISSUE TITLE
<iframe> with no `title` should use ancestor element's `title`

### ISSUE THREAD

**Issue by maxlever** [dequelabs/axe-core#2071]
Iframes with no title but with an ancestor with a title are marked as errors, yet the HTML spec says that `title` can be inherited from parent: https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute:
> If this attribute is omitted from an element, then it implies that the title attribute of the nearest ancestor HTML element with a title attribute set is also relevant to this element. 

In my particular case, `<amp-iframe>`s should be able to apply its title to `<iframe>`s without having to duplicate the attribute. Please let me know if I am mistaken about the specification. 

I am aware that the [axe docs](https://dequeuniversity.com/rules/axe/3.0/frame-title) indicate that iframes should always have an explicit title and doesn't allow ancestor titles to apply. Would love to discuss.

#### Provide the steps to reproduce
1. Run lighthouse/axe on https://www.axios.com/carbon-emissions-rate-slower-2019-f235af5a-fbd8-46fc-a196-228233f20f16.html, or on any snippet containing, for example
```html
<div title="remote kitten">
  <iframe src="http://placekitten.com/100" ></iframe>
</div>
```
2. Observe that there is an A11Y error saying that the `<iframe>` does not have a `title` attribute, and yet its <amp-iframe> parent has a `title` attribute should be applied to it.

#### What is the current behavior?
iframes that are missing an explicit `title` but have a parent with a `title` are marked as missing a `title`.

#### What is the expected behavior?
any iframe with a parent `title` should be marked as valid.

#### Environment Information
All environments

#### Related issues
https://github.com/ampproject/amphtml/issues/17308#issuecomment-426399152

#### Note
This is an cross-post from https://github.com/GoogleChrome/lighthouse/issues/10406

**Comment by WilcoFiers** [comment]
@maxlever The HTML spec says the title is "relevant", but not that it is inherited. This is not how accessible name computation works. You can see this if you look up the accessible name of the iframe element (using Chrome DevTools for example).

What I think "relevant" means in this context is something like this:

```html
<abbr title="Hypertext Transport Protocol">
  <i>HTTP<i>
</abbr>
```
In this scenario, the title is still considered relevant for the text inside the `i` element. 

Hope that helps! I don't think there is a property you can set on a parent that effects the accessible name of a child. Not sure how you'd do that, you may really have to edit that iframe element. If you have any further questions, feel free to open a new issue or reopen this one if it is relevant.
